test-suite/src/test/scala/org/scalajs/testsuite/compiler/RegressionTest.scala

```diff
@@ -432,5 +432,11 @@ object RegressionTest extends JasmineTest {
       }
       expect(x).toEqual(5)
     }
+
+    it("null.asInstanceOf[Unit] should succeed - #1691") {
+      def getNull(): Any = null // s
```

Comment:
What does this comment mean? `// s` Or is it just something that should not be there.